src/Apps/Legion/Components/ConsumeShell/ConsumeShell.types.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface IConsumeShellProps {}
+
+export type TConsumeShellClassNames =
```

Comment:
I'm probably just missing it, but where does this get used?

Reply:
This is the equivalent of `IComponentNameStyles` it gives the style function the keys to the styles you want (i.e., root). `makeStyles<IComponentClassNames>` makes sure you don't miss any class name.

Reply:
hm ok. I just wasn't seeing where it was used. I must have missed it.